zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/Keyspace.scala

```diff
@@ -24,31 +24,34 @@ import com.twitter.util.Duration
 import java.net.InetSocketAddress
 
 object Keyspace {
-  def zookeeperServerSets(
```

Comment:
why remove this code, why not create a new method that does the not-zookeeper server sets?